### PR TITLE
Don't install the bootstrap gem unless they are using sassc

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: lib/generators/blacklight/assets/importmap_generator.rb [hunk 1]
```diff
@@ -45,27 +40,38 @@ def append_blacklight_javascript
       end
 
       def add_stylesheet
-        if File.exist? 'app/assets/stylesheets/application.bootstrap.scss'
-          if ENV['CI']
-            run "yarn add file:#{Blacklight::Engine.root}"
-          else
-            run "yarn add blacklight-frontend@#{Blacklight::VERSION}"
-          end
+        return generate_with_sassc_rails unless used_bootstrap_css?
```
Comment (jrochkind) [inline lib/generators/blacklight/assets/importmap_generator.rb:43]: I would find this more readable if it were

```
unless used_bootstrap_css?
   generate_with_sassc_rails
   return
```

(Unless you actually want to use the return value of `generate_with_sassc_rails`). 

This code is getting pretty convoluted and branching, keeping it readable seems good, what do you think?

Comment (jcoyne) [reply lib/generators/blacklight/assets/importmap_generator.rb]: Sure, done.

**Comment by jrochkind** [comment]
I _think_ this makes sense (and is similar to what I ended up doing in a version in blacklight_range_limit investigating this stuff). I am interested in if it breaks #3360 if we merge that first. 

At some point I'd be interested in adding a switch so the file does not NEED to be named `application.bootstrap.scss` -- there are cases where you are using a package.json-based solution but didn't actually _generate_ with `--css=bootstrap`, and you have an `application.[s]css` that you'd still want the `@import 'blacklight'` added to. 

But that's a separate thing, this PR doesn't change it. I'll approve, but what if we merge #3360 first and rebase this one to make sure they don't conflict?

**Comment by jcoyne** [comment]
Rebased

**Review by jrochkind — APPROVED** [review]

**Pull request merged**